app/pages/sections/order-items/catalogue-solutions/order-item/flat/patient/ui-tests/general.ui.test.js

```diff
@@ -37,7 +37,6 @@ const catalogueSolutionIdInSession = 'solution-1';
 const deliveryDateInSession = '2020-10-10';
 const recipientsInSession = JSON.stringify(recipients);
 const selectedRecipientsInSession = JSON.stringify(selectedRecipients);
-
```

Comment:
I don't think we need to remove this blank line.

Reply:
added blank line